src/combinatorics/bigraph.lean

```diff
@@ -0,0 +1,289 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Aaron Anderson, Jalex Stark.
+-/
+import algebra.big_operators
+
+/-!
+# Bigraphs
```

Comment:
The term "bipartite graph" should appear somewhere here. Is bigraph a standard abbreviation? The [wikipedia page](https://en.wikipedia.org/wiki/Bigraph) is about something else.

Reply:
It pops up, for instance: https://mathworld.wolfram.com/Bigraph.html. I had not heard of the concept on that wikipedia page.

I got the term from page 27 of this model-theory paper https://www.math.ucla.edu/~matthias/pdf/VC%20Density,%20I.pdf
They do cite combinatorialists for that distinction, although I haven't checked out the cited paper yet.

The distinction is basically that a `bigraph` has a specified left part and right part, while a bipartite graph just can be partitioned... in my mind, a bipartite graph is just a graph with no odd cycles.

Reply:
Ah, I see. All this is good info to include in the module doc!

Reply:
I put a little discussion in the module docstring, without renaming the file or the declarations. I'm happy to call everything `bipartite_graph` instead if that is preferred.

Reply:
I'm fine with these being called `bigraph`s . I do think the part about there being a specified left and right part is important, so it should also be added to the module docs (and the docstrings when you get to them).

If you can add some explicit citations for this concept / name that'd be great too.